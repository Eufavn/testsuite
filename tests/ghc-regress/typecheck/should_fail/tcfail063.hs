-- !!! no type variable on a context
-- !!! reported by Sigbjorn Finne

moby :: Num => Int -> a -> Int
moby x y = x+y
