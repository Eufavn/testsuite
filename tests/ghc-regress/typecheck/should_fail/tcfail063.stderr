
tcfail063.hs:6:8:
    Kind error: `Num' is not applied to enough type arguments
    In the type signature: moby :: (Num) => Int -> a -> Int
